bgpd/bgp_attr.c

```diff
@@ -1255,6 +1254,7 @@ void bgp_attr_flush(struct attr *attr)
 		bgp_attr_set_vnc_subtlvs(attr, NULL);
 	}
 #endif
+	bgp_attr_extra_free(attr);
```

Comment:
This seems to have some problem.
You are releasing the memory of attr->extra after the memory of attr is already freed.
Check Line no. 1198

Reply:
It's flush vs. unintern.